### ISSUE TITLE
'cd' in the basic tutorial results in a syntax error

### ISSUE THREAD

**Issue by feuerbach** [YosefLab/scVI#462]
When I load `tests/notebooks/basic_tutorial.ipynb` in jupyter and run the very first cell

```
cd ../../
%matplotlib inline
```

I get

```
  File "<ipython-input-1-a8896822abed>", line 1
    cd ../../
        ^
SyntaxError: invalid syntax
```

If I split that cell into two cells, one with cd and one with matplotlib, then they both work. This may apply to other notebooks as well.

A tangentially related issue is that if I run several notebooks in the same session, then the second and subsequent `cd`s take me to the wrong directories (as I'm already in scVI's root directory after the first `cd`). A possible (but tedious) solution would be not to `cd` at all but to use relative paths.

Jupyter information:

The version of the notebook server is: 6.0.1-5fe194d
The server is running on this version of Python:

Python 3.7.4 (default, Jul  9 2019, 16:32:37) 
[GCC 9.1.1 20190503 (Red Hat 9.1.1-1)]

Current Kernel Information:

Python 3.7.4 (default, Jul  9 2019, 16:32:37) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.7.0 -- An enhanced Interactive Python. Type '?' for help.

**Comment by romain-lopez** [comment]
Hi, 

I agree this is far from ideal. For the syntax error, I believe it indeed depend on your version of ipython. We will try to fix this issue. 

Thanks for posting !
Romain

**Comment by adamgayoso** [comment]
this was fixed in #479

**Comment by Arif7067** [comment]
cd models/research/
       ^
SyntaxError: invalid syntax

How to fix this ? If needed to change the Python version then which version shall I use?
